src/node_i18n.cc

```diff
@@ -510,67 +510,6 @@ void ICUErrorName(const FunctionCallbackInfo<Value>& args) {
                           NewStringType::kNormal).ToLocalChecked());
 }
 
-#define TYPE_ICU "icu"
-#define TYPE_UNICODE "unicode"
-#define TYPE_CLDR "cldr"
-#define TYPE_TZ "tz"
-
-/**
- * This is the workhorse function that deals with the actual version info.
- * Get an ICU version.
- * @param type the type of version to get. One of VERSION_TYPES
- * @param buf optional buffer for result
- * @param status ICU error status. If failure, assume result is undefined.
- * @return version number, or NULL. May or may not be buf.
- */
-const char* GetVersion(const char* type,
```

Comment:
would it be better to keep these functions exported from node_i18n and just call them from node_metadata.cc ? (same for keeping function in node_crypto.cc ) 

LGTM with the suggestion

Reply:
I tried with that direction before, but it limits the number of includes better if `node_metadata.cc` only reference these external headers directly instead of including `node_something.h` - as those headers often include a bunch more, which sort of result in everybody including everybody...since the version computation is only ever used in `node_metadata.cc` and don't really have anything to do with other functionality code, I think it makes sense to put them directly there instead of putting them in headers.